Converted project & site dashboard to just dashboard (#1102)
blutui/dev.blutui.com: content/guides/add-recaptcha-3-to-a-form.mdx

<Steps>
<Step>
### Navigate to settings

> Head to your Blutui Dashboard. Click on **Settings** and navigate to **Forms** page.

Reviewer: ```suggestion
Head to the Dashboard. Click on **Settings** and navigate to **Forms** page.
```
